extensions/wikia/Chat2/Chat.class.php

```diff
@@ -243,7 +243,7 @@ public static function addConnectionLogEntry() {
 				'cuc_title' => 'Chat',
 				'cuc_minor' => 0,
 				'cuc_user' => $wg->User->getID(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app